lumigator/python/mzai/backend/services/experiments.py

```diff
@@ -135,10 +135,9 @@ def create_experiment(self, request: ExperimentCreate) -> ExperimentResponse:
         else:
             worker_gpus = float(os.environ.get(settings.RAY_WORKER_GPUS_ENV_VAR, 1.0))
 
-        # NOTE: Whenever the list of libraries below change, the LD_PRELOAD line in
-        # `docker-compose-local.yaml` has to be updated with a new hash value
         runtime_env = {
-            "pip": ["nltk==3.8.1", "lm-buddy[jobs]==0.12.1"],
+            "pip": ["wandb", "evaluate","pydantic_yaml", "torch", "peft","s3fs","loguru","mistralai","openai","lm-eval","langchain_community","langchain_openai","ragas","trl"],
```

Comment:
this is fairly clunky at this point - we can pass a requirements.txt file, and I know we need to add nltk back in, but I was wondering if people thought we should have these reqs as part of the base requirements of lumigator or keep them separately specifically for this module.

Reply:
I also notice a lot of calls we're doing for components I don't think we need anymore, including langchain and ragas here

Reply:
Note that we'll no longer need `"lm-buddy[jobs]==0.12.1"` 🎉

Reply:
Yup, all good points. I'll use a lm-buddy requirements.txt at the root of the repo, and in the next PR remove some of them.